ecommerceproj/theretailerapp/migrations/0018_order_status.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.8 on 2021-11-30 08:41
+
+from django.db import migrations, models
```

Comment:
migrations to be cleaned up.

- `migrate` down to `zero`
- remove all files under `/migrations`
- run `makemirations`